http/headers/headers.go

```diff
@@ -0,0 +1,7 @@
+package headers
```

Comment:
Can you elaborate why this and the mediatypes package need to be separate packages and their constants need to be exported publicly? The data provided is not specific to this package and outside code likely should not rely on them.

Next: why do these need to be separate packages from each other? For the purpose of package selenium, they both have to do with HTTP headers.

I would prefer, in order, to:

1) Move this data to package selenium in remote.go
2) Move these packages into the internal directory and merge them together.

Reply:
outdated: I've removed `http` package